### ISSUE TITLE
Country instance methods missing

### ISSUE THREAD

**Issue by labe** [mobi/telephone_number#32]
given `phone_object = TelephoneNumber.parse("3175082237", :us)`,
```
phone_object.country #=> NoMethodError
phone_object.country_data #=> NoMethodError
```
The data is still accessible but must be called via:
```
phone_object.phone_data.country_data[:id] #=> "US"
phone_object.phone_data.country_data #=> {Hash}
```

Could you please update either the code or the documentation?

**Comment by akfernun** [comment]
Hi @labe, thank you for bringing this to our attention. 

#31 should address these issues. I plan on merging and releasing that sometime next week.

**Comment by akfernun** [comment]
Hi, @labe, I wanted to make sure I followed up with you on this. I merged and cut a new release for #31. Because there are a number of internal changes in that request, I released it as a release candidate. Once it spends a couple of more days in our production environment and we're sure that there are no major bugs, I will cut an official version. With these changes, you will be able to say something like `TelephoneNumber.parse(...).country` and be able view many different aspects about the country. Once I release an official version with this functionality, I will update the documentation accordingly. Thank you.
